Common/UnitDefinitions/Energy.json

```diff
@@ -188,6 +188,18 @@
           "AbbreviationsForPrefixes": { "Deca": "Английский декатерм" }
         }
       ]
+    },
+    {
+      "SingularName": "HorsePowerHour",
+      "PluralName": "HorsePowerHour",
```

Comment:
`HorsePowerHours` in plural form, maybe?

If you change it, you also need to re-run `generate-code.bat`.

Reply:
Also it seems like the naming is lower-case `power`, as in `HorsepowerHour`.

https://en.wikipedia.org/wiki/Horsepower-hour

Reply:
done